### PR TITLE
add timeout respect to resource quota check

### PR DESCRIPTION
**Pull request by jonnydawg**

### CODE REVIEW

File: cmd/resource-quota-check/resource-quota.yaml [hunk 1]
```diff
@@ -6,19 +6,17 @@ metadata:
   namespace: kuberhealthy
 spec:
   runInterval: 30m
-  timeout: &resource_quota_check_timeout 5m
+  timeout: 5m
   podSpec:
     containers:
     - name: resource-quota
-      image: kuberhealthy/resource-quota-check:v1.1.1
+      image: kuberhealthy/resource-quota-check:v1.2.2
```
Comment (joshulyne) [inline cmd/resource-quota-check/resource-quota.yaml:13]: can this just be 1.2.0?

File: cmd/pod-restarts-check/pod-restarts-check.yaml [hunk 2]
```diff
@@ -16,11 +16,9 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-          - name: CHECK_POD_TIMEOUT
-            value: "10m"
           - name: MAX_FAILURES_ALLOWED
             value: "10"
-        image: kuberhealthy/pod-restarts-check:v2.1.1
+        image: kuberhealthy/pod-restarts-check:v2.3.0
```
Comment (joshulyne) [inline cmd/pod-restarts-check/pod-restarts-check.yaml:21]: same here.. 2.2.0? sorry, being nitpicky...

File: cmd/resource-quota-check/resource-quota.yaml (hunk 1 above)
Comment (jonnydawg) [reply cmd/resource-quota-check/resource-quota.yaml:13]: https://hub.docker.com/repository/registry-1.docker.io/kuberhealthy/resource-quota-check/tags?page=1 1.2.1 was built and pushed a while ago

File: cmd/pod-restarts-check/pod-restarts-check.yaml (hunk 2 above)
Comment (jonnydawg) [reply cmd/pod-restarts-check/pod-restarts-check.yaml:21]: [same with 2.2.1 here](https://hub.docker.com/repository/registry-1.docker.io/kuberhealthy/pod-restarts-check/tags?page=1)

**Comment by joshulyne** [comment]
just edited the resource-quota check and pod-restarts readme. looks good 👍

**Comment by integrii** [comment]
@jonnydawg Some conflicts came up....

**Pull request merged**